.changeset/cyan-kangaroos-hope.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fix false-positives for at-rule descriptors in `declaration-property-value-no-unknown`
```

Comment:
```suggestion
Fixed: `declaration-property-value-no-unknown` false positives for descriptors
```

nit for [conventions](https://stylelint.io/maintainer-guide/pull-requests).

Reply:
I'd still explicitly mention "at-rules", since most people will probably not know what "descriptors" are.

```suggestion
Fixed: `declaration-property-value-no-unknown` false positives for at-rule descriptors
```